src/calculateRentalCost.js

```diff
@@ -1,10 +1,18 @@
-/**
- * @param {number} days
- *
- * @return {number}
- */
 function calculateRentalCost(days) {
-  // write code here
+  const PRICE_ONE_DAY = 40;
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+  const SHORT_TERM_DISCOUNT = 20;
+  const SHORT__TERM = 3;
```

Comment:
There is a typo in the constant name `SHORT__TERM`. It should be `SHORT_TERM` to follow the naming convention and avoid confusion.